### PR TITLE
Improvement/use ed25519 temporary key pair

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request modifies the `packer` configuration to use an ED25519 temporary key pair instead of the default RSA key pair.

__Please note that #98 must be merged before this pull request.  This branch must also be rebased after merging #98.__

## 💭 Motivation and context ##

Linux distributions are starting to disallow the use of RSA keys by default, so it makes sense to use an ED25519 key instead.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [ ] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: src/packer.pkr.hcl [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    Application        = "Example"
+    Base_AMI_Name      = data.amazon-ami.debian_bullseye.name
+    GitHub_Release_URL = var.release_url
+    OS_Version         = "Debian Bullseye"
+    Pre_Release        = var.is_prerelease
+    Release            = var.release_tag
+    Team               = "VM Fusion - Development"
+  }
+  # Many Linux distributions are now disallowing the use of RSA keys,
+  # so it makes sense to use anED25519 key instead.
```
Comment (mcdonnnj) [inline src/packer.pkr.hcl]:
```suggestion
  # so it makes sense to use an ED25519 key instead.
```

**Review by mcdonnnj — APPROVED** [review]
One suggestion but otherwise this makes sense.

**Review by mcdonnnj — COMMENTED** [review]
https://github.com/cisagov/skeleton-packer/compare/improvement/use_hcl2...improvement/use-ed25519-temporary-key-pair is helpful for reviewing this PR.

**Review by dav3r — APPROVED** [review]
Progress! 👍

**Pull request merged**